build.sbt

```
@@ -1,23 +1,16 @@
 import sbtcrossproject.CrossPlugin.autoImport.{crossProject, CrossType}
```

```suggestion
import sbtcrossproject.CrossPlugin.autoImport.{crossProject, CrossType}
```

Auto-imports are, auto-imported :)